app/containers/InterventionDetailsPage/containers/ParticipantInviter/InviteParticipantsModalContent.tsx

```diff
@@ -81,7 +84,7 @@ export const InviteParticipantsModalContent: FC<Props> = ({
     }
   }, [organizationId]);
 
-  const sessionOptions: SelectOption<string>[] = useMemo(() => {
+  const sessionOptions: SelectOptionTypeExtended<string>[] = useMemo(() => {
```

Comment:
I see that you extended the SelectOption with `type` props. After quick look I see that is only used for filtering out options in `app/containers/InterventionDetailsPage/containers/ParticipantInviter/InviteParticipantsModalContent.tsx`. Is that right? If yes, in my opinion, creating new type in `app/components/Select/types.ts` to filter options before passing in to the Select is not the best idea. It adds additional complexity to the Select component without any Select component internal usage for it. 
I recommend either creating just two arrays of session options or doing sth like that on this line:
```suggestion
  const sessionOptions: (SelectOption<string> & { type: string })[] =
```
and reverting any changes related to `SelectOptionTypeExtended`